front/app/modules/commercial/project_folders/citizen/components/ProjectFolderSiteMap/index.tsx

```diff
@@ -29,38 +29,41 @@ const publicationStatuses: PublicationStatus[] = [
   'draft',
 ];
 
-const ProjectFolderSitemap = ({ adminPublication, hightestTitle }: Props) => {
+const ProjectFolderSitemap = ({ projectFolderId, hightestTitle }: Props) => {
   const TitleComponent = hightestTitle === 'h3' ? H3 : H4;
 
-  const childProjects = useAdminPublicationChildren({
-    publicationId: adminPublication.id,
+  const folder = useProjectFolder({ projectFolderId });
+  const { list: childAdminPublications } = useAdminPublications({
```

Comment:
Imo, it doesn't make sense to pass adminPublication when we're certain we're in a project folder. Trying to be as specific as possible for code readability.

Besides, the different ids are confusing. On admin publication you have the normal id and the publicationId (which looks as if it's the id of the admin publication and shouldn't be there, but that's for a different PR).